### ISSUE TITLE
Unknown prop 'classes'?

### ISSUE THREAD

**Issue by tsneed290** [chadly/react-bs-notifier#39]
My tests recently started failing on the following warning:

```
console.error node_modules/fbjs/lib/warning.js:33
      Warning: Unknown prop `classes` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
          in div (created by TransitionGroup)
          in TransitionGroup (created by AlertList)
          in Transition (created by CSSTransition)
          in CSSTransition (created by AlertTransition)
          in AlertTransition (created by Jss(AlertTransition))
          in Jss(AlertTransition) (created by AlertContainer)
          in div (created by TransitionGroup)
          in TransitionGroup (created by AlertContainer)
          in div (created by AlertContainer)
          in AlertContainer (created by Jss(AlertContainer))
          in Jss(AlertContainer) (created by AlertList)
          in AlertList (created by Jss(AlertList))
```

I haven't changed any of the code on my side and it was not failing last week. Any idea as to the cause?

Thanks!

**Comment by frankolson** [comment]
I am getting this same error message.

**Comment by frankolson** [comment]
I just created a PR for this that should fix this issue.

**Comment by chadly** [comment]
Thanks, @frankolson! Fixed in [`v4.4.3`](https://github.com/chadly/react-bs-notifier/releases/tag/v4.4.3%2B1) via #40
